src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -1297,7 +1339,7 @@ def forward(
 
             hidden_states = backbone_hidden_states
 
-        hidden_states = self.neck(hidden_states)
+        hidden_states = self.neck(hidden_states, None)
```

Comment:
Let's pass these in a keyword arguments so it's clearer what each value represents